www/source/blog/2018-06-13-base-plans-refresh.html.md

```diff
@@ -0,0 +1,170 @@
+   widget_world: WARN surprising order. Worse, if both versions of `acme/A` are
+   widget_world: WARN ABI-incompatible, runtime segmentation faults are more than likely.
+   widget_world: WARN
+   widget_world: WARN In order to preserve reliability at runtime the duplicate dependency
+   widget_world: WARN entries will need to be resolved before this Plan can be built.
+   widget_world: WARN Below is an expanded graph of all `$pkg_deps` and their dependencies
+   widget_world: WARN with the problematic lines noted.
+   widget_world: WARN
+   widget_world: WARN Computed dependency graph (Lines with '*' denote a problematic entry):
+
+nellshamrell/widget_world/1.0.0/<timestamp>
+    nellshamrell/widget/1.0.0/<timestamp> (*)
+        core/glibc/2.27/<timestamp> (*)
+    nellshamrell/world/1.0.0/<timestamp> (*)
+        core/glibc/2.22/<timestamp> (*)
+
+ERROR: Computed runtime dependency check failed, aborting
+```
+
+This error occurs because nellshamrell/widget_world has one dependency that was built with 2.27, and another that was built with 2.22.  This makes them uncompatible, and the nellshamrell/widget_world plan will not build.
+
+### How do I resolve this?
+
+In this case, you can resolve this error by rebuilding nellshamrell/world with core/glibc version 2.27.  If we do that, and now both dependencies depend on the same version of glibc, then nellshamrell/widget_world will build fine.
+
+```
+nellshamrell/widget_world
+-> nellshamrell/widget
+   --> core/glibc/2.27
+-> nellshamrell/world
+   --> core/glibc/2.27
+```
+
+Success!
+
+```
+(studio) $ build widget_world
+...
+Success!
+I love when a plan.sh comes together!
+```
+
+So...after the core plans refresh gets added to production, and all of the base plans and dependent base plans are promoted to stable, you may see some of these failures with your own plans.  It's also possible they may fail on more than one dependency being out of sync.  You can fix these errors in 2 ways:
+1) Rebuilding a dependency with an older version of a base plan (i.e. nellshamrell/world with core/glibc/2.22) with a newer version of that base plan, or at least the same version as the rest of your dependencies (i.e. nellshamrell/world with core/glibc/2.27).
+2) Pinning both dependencies to an older version of the base plan (i.e. we could pin both nellshamrell/widget and nellshamrell/world to core/glibc/2.22, and nellshamrell/widget_world would build just fine).
+
+### Do I need to do this for all my plans?
+
+If you have plans that depend on base plans (which I expect most if not all plans do), you will likely run into this dependency error on at least one of your plans.  You can fix it as outlined above.  If you need assistance, have questions, or just want some company through the updating process, please feel free to ask for help in the [Habitat Slack](http://slack.habitat.sh/) - we are there during normal business hours and happy to help!
```

Comment:
Is it worth calling out that you can click `Build latest version` in Builder (and then promote to stable)?

Reply:
Fixed!